documentation/modules/auxiliary/scanner/mssql/mssql_login.md

```diff
@@ -0,0 +1,43 @@
+## Vulnerable Application
+
+This module simply queries the MSSQL instance for a specific user/pass (default is sa with blank). 
+
+## Verification Steps
+Example steps in this format (is also in the PR):
+
+1. Start msfconsole
+2. Do: ```use scanner/mssql/mssql_login```
+3. Do: ```set RHOSTS [IP]```
+4. Do: ```run```
+5. You should get a shell.
+
+## Options
+A number of options interesting options without default values exist. They are as follows: 
+   
+   Name                 Current Setting  Required  Description
+   ----                 ---------------  --------  -----------
+   PASSWORD                              no        A specific password to authenticate with
+   PASS_FILE                             no        File containing passwords, one per line
+   RHOSTS                                yes       The target host(s),
+   USERPASS_FILE                         no        File containing users and passwords separated by space, one pair per line
+   USER_FILE                             no        File containing usernames, one per line
+
+Options ending in _FILE are used by specifing a file location. For example, specifying the USER_FILE would happend as follows:
+
+```
+msf6 auxiliary(scanner/mssql/mssql_login) > set USER_FILE ./userfile
+```
+
+
```

Comment:
If the module using any unique options it's worth adding an specific markdown header for each one, like so:

```suggestion
## Options

### USER_FILE

File containing users, one per line.

### PASS_FILE

File containing passwords, one per line
```

The above format is the standard for the `##Options` section (older modules weren't always written using this format FYI). Printing out the module options like you have done here is great to add and is typically included in the `##Scenarios` section of the documentation. When you're demonstrating how to run the module in `##Scenarios` set all the datastore options required, the run the `options` command to demonstrate to the user what must be set to run the module successfully then copy and paste the entire output into `##Scenarios`